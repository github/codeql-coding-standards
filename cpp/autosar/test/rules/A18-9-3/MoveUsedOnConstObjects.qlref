rules/A18-9-3/MoveUsedOnConstObjects.ql