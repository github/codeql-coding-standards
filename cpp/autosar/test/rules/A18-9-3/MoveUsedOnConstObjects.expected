| test.cpp:15:20:15:21 | a1 | Implicit call to copy constructor use in std::move because argument has const type `const A`. |
| test.cpp:17:18:17:19 | a3 | Implicit call to copy constructor use in std::move because argument has const type `const A &`. |
